snc/redis-bundle/2.0/etc/packages/snc_redis.yaml

```diff
@@ -0,0 +1,6 @@
+snc_redis:
+    clients:
+        default:
+            type: predis
+            alias: default
+            dsn: "%env(REDIS_URL)%"
```

Comment:
IIRC correctly, SncRedisBundle does not support configuring the DSN through a single environment variable.